tests/conftest.py

```diff
@@ -14,6 +15,30 @@
 FIXTURES_DIR = join(TESTS_DIR, 'fixtures')
 CASSETTE_DIR = join(FIXTURES_DIR, 'vcr_cassettes')
 
+try:
```

Comment:
This is already pretty clean, but we could also use [`pytest.importorskip`](https://docs.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency) on the test functions.

Reply:
Ah, that's perfect. Thanks for the suggestion!